services/masterdata/src/lib/core/ui/masterdata-type-selector/masterdata-type-selector.component.ts

```diff
@@ -15,7 +15,7 @@ import { MasterdataType } from "../../models";
 	]
 })
 export class MasterdataTypeSelectorComponent implements ControlValueAccessor {
```

Comment:
Please take a look in the updated autocomplete component. Its more usable now since its based on the key and not the whole KeyValuePair<T,T>. Note that this is currently in PR62